apps/src/lib/levelbuilder/lesson-editor/FindProgrammingExpressionDialog.jsx

```diff
@@ -203,6 +199,14 @@ class FindProgrammingExpressionDialog extends Component {
           handleSelect={this.handleSelectProgrammingExpression}
         />
 
+        <div style={{padding: 20}}>
+          <PaginationWrapper
```

Comment:
It occurs to me that the `fetch` logic should probably have originally been put into the `ProgrammingExpressionTable` component itself; that would mean that we could also put the `PaginationWrapper` in there and have the whole thing be much more self-contained.

That being said, I don't think that refactor is necessary for this PR but it would be good to track as follow-up work